client/dwh_migration_client/gcs_util.py

```diff
@@ -71,11 +72,11 @@ def download_directory(local_dir: str, bucket_name: str, gcs_path: str) -> None:
     logging.info('Start downloading outputs from gcs "%s/%s"', bucket_name, gcs_path)
     for blob in blobs:
         file_name = basename(blob.name)
-        sub_dir = blob.name[len(gcs_path) + 1 : -len(file_name)]
-        file_dir = join(local_dir, sub_dir)
+        sub_dir = Path(blob.name[len(gcs_path) + 1 : -len(file_name)])
+        file_dir = Path(local_dir) / sub_dir
         os.makedirs(file_dir, exist_ok=True)
-        file_path = join(file_dir, file_name)
+        file_path = file_dir / file_name
         logging.info('Downloading output file to "%s"...', file_path)
         blob.download_to_filename(file_path)
 
-    logging.info('Finished downloading. Output files are in "%s".', local_dir)
+    logging.info('Finished downloading. Output files are in "%s".', local_dir)
```

Comment:
Maybe run `black` which would ensure a newline here. You ca run all the static analysis tools via :`cd client && make check`

Reply:
Done. Thanks